CONTRIBUTING.md

```diff
@@ -182,3 +182,10 @@ public RealmQuery<E> equalTo(String fieldName, String fieldValue, boolean caseSe
 
 Above is based on the official guidelines from Oracle regarding Javadoc: http://www.oracle.com/technetwork/articles/java/index-137868.html
 
+### Branch Strategy
+
+* `releases` branch is where the hotfix versions released from. Releases from this branch will bump the last part of the version number.
+* `master` branch is where the major/minor versions released from. The first or the second part of the version number will be bumped when release from this branch.
```

Comment:
`master` is the main development branch for new features and breaking changes. It also where major/minor versions are released from. The major or minor version number will be bumped when releasing from this branch.